e2e-tests/batch-mode-tests.sh

```diff
@@ -69,6 +69,29 @@ function setup() {
 }
 
 
+#######################################
+# Function to use count resources in openmrs server
+# Arguments:
+#   directory to sink files to
```

Comment:
Thanks for refactoring this function. Please add the global variables that this function sets (e.g., `total_test_patients`) in the `Globals:` section of function doc ([style guide](https://google.github.io/styleguide/shellguide.html#s4.2-function-comments)).